### PR TITLE
bind dev permissions to ids

### PR DESCRIPTION
**Pull request by Igor-aka-Gillirond**

### CODE REVIEW

File: src/dev-permissions.js [hunk 1]
```diff
@@ -7,109 +7,109 @@ const AccessTypes = {
...
+  APP_RESET                 : 13, //Application reset
+  MODIFY_CORS_DOMAIN_CONTROL: 8, //Modify CORS/domain control
+  MODIFY_CUSTOM_DOMAIN      : 9, //Modify custom domain
+  MODIFY_EMAIL_SETTINGS     : 7, //Modify email settings
+  MODIFY_LIMIT_NOTIFICATION : 14, //Modify limit notification
+  MODIFY_MOBILE_SETTINGS    : 6, //Modify mobile settings
+  MODIFY_SOCIAL_SETTINGS    : 5, //Modify social settings
+  DELETE_APPLICATION        : 11, //Delete application
+  ENABLE_DISABLE_GIT        : 10, //Enable/disable git
+  INSTALL_LICENSE           : undefined, //TODO Install license
```
Comment (stanislav-grin) [inline src/dev-permissions.js:23]: why undefined here?

File: src/dev-permissions.js [hunk 2]
```diff
@@ -7,109 +7,109 @@ const AccessTypes = {
...
+  MANAGE_API_KEYS           : 4, //Manage API keys
+  PROMOTE_APPLICATION       : undefined, //Promote schema from one of your applications to another
```
Comment (stanislav-grin) [inline src/dev-permissions.js:25]: ..and here

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backendless-console-sdk",
-  "version": "2.10.8",
+  "version": "2.10.9",
```
Comment (stanislav-grin) [inline package.json:3]: Don't change version here in this PR, it should be changed **after** this PR is merged.

Comment (Igor-aka-Gillirond) [reply package.json]: rolledback

File: src/dev-permissions.js [hunk 4]
```diff
@@ -7,109 +7,107 @@ const AccessTypes = {
...
+  APP_RESET                 : 13, //Application reset
+  MODIFY_CORS_DOMAIN_CONTROL: 8, //Modify CORS/domain control
+  MODIFY_CUSTOM_DOMAIN      : 9, //Modify custom domain
+  MODIFY_EMAIL_SETTINGS     : 7, //Modify email settings
+  MODIFY_LIMIT_NOTIFICATION : 14, //Modify limit notification
+  MODIFY_MOBILE_SETTINGS    : 6, //Modify mobile settings
+  MODIFY_SOCIAL_SETTINGS    : 5, //Modify social settings
+  DELETE_APPLICATION        : 11, //Delete application
+  ENABLE_DISABLE_GIT        : 10, //Enable/disable git
+  //INSTALL_LICENSE           : undefined, //TODO Install license
```
Comment (Valodya) [inline src/dev-permissions.js:23]: @Igor-aka-Gillirond do we still need this one?

Comment (Igor-aka-Gillirond) [reply src/dev-permissions.js]: No, removed

**Pull request merged**